src/ruleconfig.jl

```diff
@@ -0,0 +1,19 @@
+"""
+    ReverseRuleConfigBackend
+
+AD backend that uses reverse mode with any ChainRules-compatible reverse-mode AD package.
+"""
+struct ReverseRuleConfigBackend{RC <: RuleConfig} <: AbstractReverseMode
+    ruleconfig::RC
+end
+
+AD.@primitive function pullback_function(ab::ReverseRuleConfigBackend, f, xs...)
+    return (vs) -> begin
+        _, back = rrule_via_ad(ab.ruleconfig, f, xs...)
+        if vs isa Tuple && length(vs) === 1
```

Comment:
Does this compile away in the same way as, I would assume, `if vs isa Tuple{Any}`?